lib/ui/window.dart

```diff
@@ -618,6 +618,16 @@ class Window {
   Size get physicalSize => _physicalSize;
   Size _physicalSize = Size.zero;
 
+  /// The physical depth is the maximum elevation that the Window allows.
+  ///
+  /// Physical layers drawn at or above this elevation will have their elevation
+  /// clamped to this value.
```

Comment:
nit: Do you think it's worth mentioning that elevation is cumulative, and that's what's going to be considered instead of what's actually set on a particular entity?

Reply:
Added a line - does that sound right?

Reply:
LG, thanks!